databases/week1/OriginalAndNewHomework-Week1.sql

```diff
@@ -86,7 +86,4 @@ FROM user
 INNER JOIN task ON task.user_id = user.id
 INNER JOIN STATUS ON STATUS.id = task.status_id
 WHERE STATUS.NAME = 'Done'
-GROUP BY user.id
-
-
-    
\ No newline at end of file
+GROUP BY user.id    
```

Comment:
You definitely to brush up your git and branches that you have 😅 Here you have a file for the first week of the module.

Reply:
🤦 Omg
Do you have time to help me with git? I´m also having troubles commiting the next homeworks D: